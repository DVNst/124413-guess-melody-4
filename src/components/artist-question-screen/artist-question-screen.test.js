import React from 'react';
import renderer from 'react-test-renderer';
import ArtistQuestionScreen from './artist-question-screen.jsx';

const question = {
  type: `artist`,
  song: {
    artist: `Jim Beam`,
    src: `audio_src`,
  },
  answers: [{
    picture: `avatar_url_1`,
    artist: `John Snow`,
  }, {
    picture: `avatar_url_2`,
    artist: `Jack Daniels`,
  }, {
    picture: `avatar_url_3`,
    artist: `Jim Beam`,
  }],
};

it(`Render ArtistQuestionScreen`, () => {
  const tree = renderer.create(
      <ArtistQuestionScreen
        question={question}
        onAnswer={() => {}}
        renderPlayer={() => {}}
      />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
