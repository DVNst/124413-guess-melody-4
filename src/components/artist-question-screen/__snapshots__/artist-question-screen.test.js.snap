// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render ArtistQuestionScreen 1`] = `
<section
  className="game__screen"
>
  <h2
    className="game__title"
  >
    Кто исполняет эту песню?
  </h2>
  <div
    className="game__track"
  >
    <div
      className="track"
    />
  </div>
  <form
    className="game__artist"
  >
    <div
      className="artist"
    >
      <input
        className="artist__input visually-hidden"
        defaultValue="answer-0"
        id="answer-0"
        name="answer"
        onChange={[Function]}
        type="radio"
      />
      <label
        className="artist__name"
        htmlFor="answer-0"
      >
        <img
          alt="John Snow"
          className="artist__picture"
          src="avatar_url_1"
        />
        John Snow
      </label>
    </div>
    <div
      className="artist"
    >
      <input
        className="artist__input visually-hidden"
        defaultValue="answer-1"
        id="answer-1"
        name="answer"
        onChange={[Function]}
        type="radio"
      />
      <label
        className="artist__name"
        htmlFor="answer-1"
      >
        <img
          alt="Jack Daniels"
          className="artist__picture"
          src="avatar_url_2"
        />
        Jack Daniels
      </label>
    </div>
    <div
      className="artist"
    >
      <input
        className="artist__input visually-hidden"
        defaultValue="answer-2"
        id="answer-2"
        name="answer"
        onChange={[Function]}
        type="radio"
      />
      <label
        className="artist__name"
        htmlFor="answer-2"
      >
        <img
          alt="Jim Beam"
          className="artist__picture"
          src="avatar_url_3"
        />
        Jim Beam
      </label>
    </div>
  </form>
</section>
`;
